Formula/gdl.rb

```diff
@@ -85,10 +85,12 @@ def install
       -lglib-2.0
       -lgobject-2.0
       -lgtk-3
-      -lintl
       -lpango-1.0
       -lpangocairo-1.0
     ]
+    on_macos do
+      flags << "-lintl"
+    end
```

Comment:
Do you prefer these one-line blocks? If you have no preference I think
```ruby
on_macos { flags << "-lintl" }
```
looks nicer. Also more consistent with linting for
```
if condition
  foo
end
```
statements.

Reply:
I have no preference, but as we started with the other syntax all over the place. I guess we should enforce it with an audit for one-line statements.

Reply:
So until we have an audit for it and update all the other statements, I'll use the full blocks as in this PR :)